docs/guide/index.rst

```diff
@@ -124,6 +124,7 @@ Additional Notes
 
    about/index
    contributing
+   community-spotlight-interviewing-guide
```

Comment:
This should go in the `organizers-guide` file. This is for staff not users.

Reply:
Does the file need to go into a folder before I create the toctree, and what do you think the depth should be?